lib/rex/post/meterpreter/channels/pool.rb

```diff
@@ -106,33 +106,32 @@ def seek(offset, whence = SEEK_SET)
         raise RuntimeError, "Invalid seek whence #{whence}.", caller
     end
 
-    request = Packet.create_request('core_channel_seek')
+    request = Packet.create_request(COMMAND_ID_CORE_CHANNEL_SEEK)
 
     request.add_tlv(TLV_TYPE_CHANNEL_ID, self.cid)
     request.add_tlv(TLV_TYPE_SEEK_OFFSET, offset)
     request.add_tlv(TLV_TYPE_SEEK_WHENCE, sane)
 
     begin
-      response = self.client.send_request(request)
+      self.client.send_request(request)
+      tell
```

Comment:
Not going to lie, I do not remember adding this change. Is this something being readded thanks to rebasing?

Reply:
It's in 8070074da39e8cb11ef3677cfa31c27117d76383

This change was in this PR before the rebase too; I had it already checked out and compared with this. So, you're not imagining things and no rebase mysteries :)

Reply:
The earlier pre-rebase commit was e81067f38f. You should still be able to see it with `git show` even after a rebase just for comparison and sanity's sake.